### DISCUSSION TITLE
Please include auto-chat in "Chat Filter" plug-in

### DISCUSSION THREAD

**Discussion by I3ulks** [runelite/runelite#15804] [Ideas]
This helps in the GE when you are filtering certain things that are being spammed.  You can filter Game chat /Public Chat/ Friends Chat /Clan chat-- adding auto-chat would be very helpful.

**Comment by Alexsuperfly** [comment]
you have to explicitly enable auto chat to be seen already, why cant you just turn that off

**Reply by I3ulks** [reply]
It's more about the interaction with another plug-in that is giving me issues -- the Chat Notifications one. 

On "Chat Notifications" plugin there is an option to ping when certain words are said. But if the phrase would normally be filtered out by the "Chat-Filter" it will still ping you if its on auto-chat. 

Example: I want a ping if someone says "Buying Logs" but not "Selling logs" so I set the plugin to notify when "Logs" is said
If you have the filter set to remove the message "Selling Logs" then you will only get pinged when someone says "buying logs" in public chat. 

However, if someone sets their autochat to "Selling Logs" that will not be filtered and you will get constant pings.
